six_patches_of_pain.go

```diff
@@ -166,11 +167,15 @@ func getGNT4ISO() string {
 			return err
 		}
 		if !info.IsDir() && isGNT4(path) {
+			// Found, stop searching by returning EOF
 			gnt4Iso = path
+			return io.EOF
```

Comment:
This fixes a small bug where other ISO files in the Six Patches of Pain directly will be checksum'd even if a valid checksum was found